dvc/parsing/__init__.py

```diff
@@ -25,6 +26,10 @@
 WDIR_KWD = "wdir"
 DEFAULT_PARAMS_FILE = ParamsDependency.DEFAULT_PARAMS_FILE
 PARAMS_KWD = "params"
+FOREACH_KWD = "foreach"
+IN_KWD = "in"
+
+DEFAULT_SENTINEL = object()
```

Comment:
Can't we use `None` instead of `DEFAULT_SENTINEL`? Seems like it's used only to check whether the key has been provided?

Reply:
Not trying to assume to much about the parametrized data.